app/models/container_image.rb

```diff
@@ -112,5 +112,9 @@ def disconnect_inv
     save
   end
 
+  def openshift_image?
+    command.present?
```

Comment:
@enoodle why `command` being present would give an indication if it's an `openshift_image` or not?